Schema/authn/002-fix-login.blue.sql

```diff
@@ -17,7 +17,7 @@ ALTER TABLE odin.login_success
         ON UPDATE NO ACTION ON DELETE NO ACTION DEFERRABLE;
 
 ALTER TABLE odin.credentials_password_ledger
-  ADD COLUMN login TEXT NOT NULL;
+  ADD COLUMN login TEXT;
```

Comment:
Probably you want a `.green.sql` that sets this to `NOT NULL`?

Reply:
Yes, let's discuss about blue-green migration tomorrow.

Reply:
I guess we can't run them separately anyway, so let's merge this?